src/axom/primal/tests/primal_polygon.cpp

```diff
@@ -369,6 +369,88 @@ TEST(primal_polygon, area_2d_3d)
   }
 }
 
+//------------------------------------------------------------------------------
+TEST(primal_polygon, normal)
+{
+  using Polygon3D = axom::primal::Polygon<double, 3>;
+  using Point3D = axom::primal::Point<double, 3>;
+
+  using Triangle3D = axom::primal::Triangle<double, 3>;
+  using Vector3D = axom::primal::Vector<double, 3>;
+
+  // Test a simple right triangle
+  {
+    Polygon3D poly3D({Point3D {0, 0, 1}, Point3D {1, 0, 0}, Point3D {1, 1, 1}});
+    Vector3D poly_normal = poly3D.normal().unitVector();
+
+    Triangle3D tri3D({Point3D {0, 0, 1}, Point3D {1, 0, 0}, Point3D {1, 1, 1}});
+    Vector3D tri_normal = tri3D.normal().unitVector();
+
+    for(int i = 0; i < 3; ++i)
+      EXPECT_NEAR(poly_normal[i], tri_normal[i], 1e-10);
+  }
+
+  // Test a planar 5-gon inscribed in a circle
+  {
+    Vector3D v1 = Vector3D({0.0, 1.0, 2.0}).unitVector();
+    Vector3D v2 = Vector3D({2.0, -1.0, 0.5}).unitVector();
+    Vector3D exp_normal = Vector3D::cross_product(v1, v2).unitVector();
+
+    Polygon3D poly(5);
+
+    double angles[5] = {0.0, 0.5, 1.2, 3.0, 5.0};
+
+    for(int i = 0; i < 5; ++i)
+      poly.addVertex(Point3D {cos(angles[i]) * v1[0] + sin(angles[i]) * v2[0],
```

Comment:
```suggestion
      {
        poly.addVertex(Point3D {cos(angles[i]) * v1[0] + sin(angles[i]) * v2[0],
```

and close it off with a vertically aligned '}'